src/Illuminate/Foundation/Testing/Wormhole.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Illuminate\Foundation\Testing;
 
-use Illuminate\Support\Facades\Date;
+use Illuminate\Support\Carbon;
```

Comment:
Change `Illuminate\Support\Facades\Date` facade to `Illuminate\Support\Carbon` here since `Date` can either have an instance of `Carbon` or `CarbonImmutable` depending on the application configuration.